nevergrad/optimization/oneshot.py

```diff
@@ -121,7 +126,8 @@ class RandomSearchMaker(base.ConfiguredOptimizer):
          - "random": uses a randomized pattern for the scale.
          - "auto": scales in function of dimension and budget (see XXX)
     recommendation_rule: str
-        "average_of_best" or "pessimistic"; "pessimistic" is the default and implies selecting the pessimistic best.
+        "average_of_best" or "pessimistic" or "average_of_meunier_best"; "pessimistic" is 
```

Comment:
Better names would be really useful, and maybe what to expect for each of these (what works best in which case?)

Reply:
Done.